.github/workflows/doc_gen.yml

```diff
@@ -27,7 +27,15 @@ jobs:
       run: |
         export LC_ALL=C.UTF-8
         export LANG=C.UTF-8
-        bundle exec jekyll build
+        OLD_IFS=$IFS
+        IFS=
+        GEN_ERRORS=$(bundle exec jekyll build 3>&2 2>&1 1>&3)
```

Comment:
The stdout must be printed and the stderr must be saved in the environment variable.